pkg/apis/networking/validation/validation_test.go

```diff
@@ -2239,8 +2242,9 @@ func TestValidateIngressUpdate(t *testing.T) {
 
 func TestValidateIngressClass(t *testing.T) {
 	testCases := map[string]struct {
-		ingressClass networking.IngressClass
-		expectedErrs field.ErrorList
+		ingressClass  networking.IngressClass
+		expectedErrs  field.ErrorList
+		enableFeature bool
```

Comment:
Nit: Tests like this sometimes end up testing multiple feature gates. In case that happens here, it would be helpful to have a more specific name like `enableNamespaceScopedParams`.